Sources/PayPalUI/UIPayPalButton.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+import SwiftUI
+
+/// Configuration for PayPal button
+public final class UIPayPalButton: PaymentButton {
```

Comment:
The name `UIPayPalButton` naming scheme is not the clearest in my opinion. Outside of this file it can be unclear whether a `UIPayPalButton` or `UIPayPalPayLaterButton` is a UIKit Button or a SwiftUI button. I can't think of an alternative though. Maybe the UIKit ones could be `PayPalUIButton`?